app/src/main/java/com/batch/mcs/finalproject/CalendarDisplayFragment.java

```diff
@@ -0,0 +1,65 @@
+package com.batch.mcs.finalproject;
+
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link CalendarDisplayFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class CalendarDisplayFragment extends Fragment {
+    // TODO: Rename parameter arguments, choose names that match
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    private static final String ARG_PARAM1 = "param1";
+    private static final String ARG_PARAM2 = "param2";
+
+    // TODO: Rename and change types of parameters
+    private String mParam1;
+    private String mParam2;
+
+
+    public CalendarDisplayFragment() {
```

Comment:
Please make the constructor the only thing here.... everything else should be not injected if not relevant